ios/FluentUI.Demo/FluentUI.Demo/Demos/BottomSheetDemoController.swift

```diff
@@ -49,12 +49,12 @@ class BottomSheetDemoController: UIViewController {
         bottomSheetViewController?.isHidden.toggle()
     }
 
-    @objc private func fullScreenExpandedOffset() {
-        bottomSheetViewController?.expandedHeightFraction = 1.0
+    @objc private func fullScreenSheetContent() {
+        bottomSheetViewController?.preferredExpandedContentHeight = 0
```

Comment:
bit counterintuitive for me that fullscreensheet is 0 contenthieght?

Reply:
Any very large number will also result in a full screen sheet, which makes more sense. 0 is just a convenient distinct default number and the desired usage is really that when a client wants a full screen sheet, they just don't set the preferredExpandedContentHeight prop at all. Is there a behavior you'd prefer for the 0 case?

Reply:
This is probably at least worth a comment in code as consumers would be using this as an example for integration. But as you mention if someone wants this behavior, the example they follow should just not set the property at all? Could add a comment saying that we're avoiding setting the property since the default value gives us full screen?

Reply:
Added a comment.